Branch-SDK/Branch-SDK/Requests/BranchShortUrlSyncRequest.m

```diff
@@ -84,7 +84,9 @@ - (NSString *)createLongUrlForUserUrl:(NSString *)userUrl {
 }
 
 + (NSString *)createLinkFromBranchKey:(NSString *)branchKey tags:(NSArray *)tags alias:(NSString *)alias type:(BranchLinkType)type matchDuration:(NSInteger)duration channel:(NSString *)channel feature:(NSString *)feature stage:(NSString *)stage params:(NSDictionary *)params {
-    NSMutableString *baseUrl = [[NSMutableString alloc] initWithFormat:@"%@/a/%@?", BNC_LINK_URL, branchKey];
+    BNCPreferenceHelper *preferenceHelper = [BNCPreferenceHelper preferenceHelper];
+    NSMutableString *baseUrl = [preferenceHelper.userUrl mutableCopy];
```

Comment:
The 2 cases I test are: getShortUrl and showShareSheet in the testbed. 
case 1: it will return a null url and the error will return "The Internet connection appears to be offline." This is the current behavior since getShortUrl hits createLongUrlForUserUrl at some point. 

case 2: It will not attach a link to the result, so if there is a share text, it will only share text.

Do you have other cases in mind that I could test?